src/pages/components.tsx

```diff
@@ -44,6 +45,16 @@ const radioSelections = [
 const Components: React.FC = () => {
   const handleClick = () => {};
 
+  const exampleSelections = [
+    { id: "option1", title: "Option 1" },
+    { id: "option2", title: "Option 2" },
+    { id: "option3", title: "Option 3" },
+  ];
+
+  const handleSelectionChange = (selected: string[]) => {
+    console.log("Selected options:", selected);
+  };
+
```

Comment:
These changes are very great for debugging purposes, but I think you forgot to update the props for the checkbox component itself.

Can update the checkbox component props and should be working great! Of course easily extensible to check for the radio button as well

Reply:
I'm not so sure what you mean by update the props inside of the component. The props for the checkbox components are not changed since they are title, the selections and the method.